wweval/R/get_input_data.R

```diff
@@ -266,17 +272,18 @@ clean_ww_data <- function(nwss_subset) {
     ungroup() |>
     rename(
       date = sample_collect_date,
-      ww = pcr_target_avg_conc,
-      ww_pop = population_served
+      site_pop = population_served
     ) |>
     mutate(
       location = toupper(wwtp_jurisdiction),
       site = wwtp_name,
-      lab = lab_id
+      lab = lab_id,
+      log_genome_copies_per_ml = log(pcr_target_avg_conc + 1e-8),
```

Comment:
Sort of random question, why not using something smaller than that? Why not 1e-20? or why not 1e-10? I don't see the place where this was done previously. There's also `.Machine$double.min.exp` (and others) for you to consider.

Reply:
Create an issue to discuss how to handle 0s in preprocessing?

Reply:
Given that we have a censoring model it _shouldn't_ ever matter. I think my instinct would be something like 

```R
log_genome_copies_per_ml = ifelse(pcr_target_avg_conf > lod_sewage, log(pcr_target_avg_conc), log(log_sewage / 2)
``` 

or similar